apps/server/src/modules/user/service/user.service.ts

```diff
@@ -9,21 +9,26 @@ import { RoleDto } from '@src/modules/role/service/dto/role.dto';
 import { RoleService } from '@src/modules/role/service/role.service';
 import { SchoolService } from '@src/modules/school';
 import { SchoolMapper } from '@src/modules/school/mapper/school.mapper';
+import { TransactionUtil } from '@shared/common/utils/transaction.util';
+import { ObjectId } from '@mikro-orm/mongodb';
 import { IUserConfig } from '../interfaces';
 import { UserMapper } from '../mapper/user.mapper';
 import { UserDto } from '../uc/dto/user.dto';
+import { AccountRepo } from '../../account/repo/account.repo';
 
 @Injectable()
 export class UserService {
 	constructor(
 		private readonly userRepo: UserRepo,
+		private readonly accountRepo: AccountRepo,
```

Comment:
```suggestion
		private readonly accountRepo: AccountRepo,
```
You should use a service

Reply:
I'm using the service now, thanks